addons/binding/org.openhab.binding.homepilot/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Can you also add the about.html, see [[Coding guidelines B.2](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)]